tests/unit/test_instance_subcmd.py

```diff
@@ -42,9 +45,9 @@
 Commands:
   associators   Get associated instances or names.
   count         Get instance count for classes.
-  create        Create an instance of classname.
+  create        Create an instance of CLASSNAME.
   delete        Delete a single CIM instance.
-  enumerate     Enumerate instances or names of CLASSNAME.
+  enumerate     Enumerate CIMInstances or names of CLASSNAME.
   get           Get a single CIMInstance.
```

Comment:
At the command level, this command gets an instance. The class CIMinstance is not known there.

Reply:
Fixed. Now says instances